src/Sylius/Behat/Context/Setup/PaymentContext.php

```diff
@@ -83,20 +83,20 @@ public function __construct(
     }
 
     /**
-     * @Given the store allows paying :paymentMethodName
-     * @Given the store allows paying with :paymentMethodName
+     * @Given the store (also) allows paying (with) :paymentMethodName
```

Comment:
After this change few steps could be broken. It is because `also` keyword is optional but spaces around it are not. So only two valid sentences are `store  allows` and `store also allows`(same for `with`)